Strain/StrainGauge.h

```diff
@@ -0,0 +1,78 @@
+#ifndef STRAIN_GAUGE_H
+#define STRAIN_GAUGE_H
+
+#include <Windows.h>
```

Comment:
**(must be changed)**: change dependency/header order.

Reply:
Done